tor_archivist/main.py

```diff
@@ -13,7 +13,16 @@
 from tor_archivist import __version__
 
 
+thirty_minutes = 1800  # seconds
```

Comment:
I like to do `60 * 30` when doing time stuff. I feel like it just makes it look cleaner.

Reply:
Funny you mention looking cleaner, that's exactly what I was trying for here. I was trying to keep it both clean and efficient by having a constant (I know it's not actually a constant) act as a label for however we use "30 minutes". Since we had it labeled, putting something a bit more complicated seemed okay if it saved a few CPU cycles.

Reply:
Yeah, the variable name makes it very clear. Sounds good then.